semver/bump.py

```diff
@@ -0,0 +1,65 @@
+from enum import IntEnum
+import subprocess, os
+from semver.logger import logging, logger, console_logger
+
+
+try: 
+    from configparser import ConfigParser
+except ImportError:
+    # Python < 3
+    from ConfigParser import ConfigParser
+
+def bump_version(version, index=2, tag_repo = True, update_files=True):
+    v = version.split('.')
+
+    # Bump version
+    v[index] = str(int(v[index]) + 1)
+
+    # Reset subversions
+    i = len(v) - 1
+    while i > index:
```

Comment:
I like this because it works for more than just major, minor and patch versions - we would just need to add to the enum and make sure we have an entry in the config file

Reply:
At first I was hard coding a patch, minor, and major, but I saw there was a lot of repetitive code so I thought this was a much more simple solution in that respect.